components/atoms/UserCard/user-card.tsx

```diff
@@ -32,9 +32,9 @@ const UserCard = ({ username, name, meta, loading }: UserCardProps) => {
               src={avatarUrl}
               alt={`${username}'s avatar image`}
             />
-            <div>
-              <h3 className="text-base text-center">{name}</h3>
-              <p className="text-center text-light-slate-9">{`@${username}`}</p>
+            <div className="text-center">
+              <h3>{name}</h3>
+              <a className="text-light-slate-9" href={`/user/${username}`}>{`@${username}`}</a>
```

Comment:
Why don't we use the `Link` component instead of using `a` directly

Reply:
I am not sure what is the advantage of using it, since the links renders to `<a>` component: 

> <Link> is a React component that extends the HTML <a> element to provide [prefetching](https://nextjs.org/docs/app/building-your-application/routing/linking-and-navigating#1-prefetching) and client-side navigation between routes. It is the primary way to navigate between routes in Next.js.

it would be valuable if we are staying in the same website to use the react features, otherwise it would add some extra complexity from my point. 

https://nextjs.org/docs/pages/api-reference/components/link

Reply:
Thank you for the feedback, I mixed this with the GitHub PRs 😭

Reply:
Done 👍